tool/gravity/cli/status.go

```diff
@@ -382,11 +382,17 @@ func printNodeStatus(node statusapi.ClusterServer, w io.Writer) {
 		fmt.Fprintf(w, "            Status:\t%v\n", color.YellowString("offline"))
 	case statusapi.NodeHealthy:
 		fmt.Fprintf(w, "            Status:\t%v\n", color.GreenString("healthy"))
+		for _, probe := range node.WarnProbes {
+			fmt.Fprintf(w, "            [%v]\t%v\n", constants.WarnMark, color.New(color.FgYellow).SprintFunc()(probe))
```

Comment:
```go
color.New(color.FgYellow).Sprint(probe)
```

Reply:
IIRC there was a reason I did it this way. Can't remember right now what it was :)

Reply:
Hm, it seems like they function the same from what I've seen. If you remember purposely doing it this way for some reason, then we'll just leave it for now.